drivers/dac/ad3552r/ad3552r.c

```diff
@@ -409,6 +409,61 @@ static int32_t _ad3552r_transfer_with_crc(struct ad3552r_desc *desc,
 	return 0;
 }
 
+#ifdef XILINX_PLATFORM
```

Comment:
so as a principle we want to avoid any driver have this kind of platform specific behavior, device driver code should not care what platform it's running on, so I suggest you move this specific behavior in drivers/platform/xilinx

Reply:
Hi, i fixed this, remove all ifdefs, leaving the sample code selecting the plpatform.
If the only platform to be supported is zedboard zynq7000, and not SDP-H1 anymore (actually reported as supported in the wiki), non xilinx functions can be removed.